sos/report/plugins/container_log.py

```diff
@@ -9,26 +9,35 @@
 # See the LICENSE file in the source distribution for further information.
 
 import os
-from sos.report.plugins import Plugin, IndependentPlugin
+from sos.report.plugins import Plugin, IndependentPlugin, PluginOpt
 
 
 class ContainerLog(Plugin, IndependentPlugin):
 
     short_desc = 'All logs under /var/log/containers'
     plugin_name = 'container_log'
     logdir = '/var/log/containers/'
+    poddir = '/var/log/pods/'
+    rotated_dirs = [poddir + '*/*.log.*', poddir + '*/*/*.log.*']
     files = (logdir, )
 
+    option_list = [
+        PluginOpt('rotated', default=False, val_type=bool,
+                  desc='also get rotated logs from /var/log/pods'),
+    ]
```

Comment:
Just noticed this on a second pass, but I'm assuming this should be dropped with the intent being that the rotated logs are collected by `-k container_log.all-logs=true` once #3681 is merged?

Reply:
I'm stupid, you are right.
This PR is changing existing behavior and it's adding a plugin option that is not being used.
I'll make the rotated log gathering conditional to the `rotated` plugin option.